npm/binary-install.js

```diff
@@ -0,0 +1,114 @@
+const { existsSync, mkdirSync, createWriteStream } = require("fs");
+const { join } = require("path");
+const { spawnSync } = require("child_process");
+
+const axios = require("axios");
```

Comment:
Why axios? Can `fetch` can be used there? I'm not care about users with node < 16 version. Well, if we care, then we should use atlease `got`, because it much better lib for making requests.

Reply:
umm i thought that's just a simple request to fetch the binary so I didn't give much attention to the lib we are using and on the other hand `axios` was quite a standard library.
but `got` can also be used if you prefer

Reply:
Well, as i said i'm prefer just native fetch, because we do not need special features here `axios` and `got` providing. But it can lock users with node < 16 version. I guess it ok to use axios, as i just checked - got esm only, which also can block something.